content/en/blog/_posts/2020-08-04-leaving-the-swarm-road-to-kubernetes.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: blog
+title: "Leaving the Swarm: The Road to Kubernetes"
+date: 2020-08-04
+slug: leaving-the-swarm-road-to-kubernetes
+url: /blog/2020/08/leaving-the-swarm-road-to-kubernetes
```

Comment:
You can probably omit this line, and leaving it out makes it less likely that a localizer will copy the value verbatim by accident (which has a 50:50 chance of replacing your English article with the problem localization).